Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Hellscream dna

## Changelog:
- Added Grommash DNA.
- Added the Long with Ponytail hair for Grommash and other orcs.

# How to test:
Check how Grommash looks in the 603 bookmark.
Check a new haircut on Grommash and other orcs.
